doc/.yarnrc.yml

```diff
@@ -1,5 +1,3 @@
 compressionLevel: mixed
 
 enableGlobalCache: false
-
-yarnPath: .yarn/releases/yarn-4.3.1.cjs
```

Comment:
If I recall correctly, Josh and I added this line when we were initially setting up the doc site on our own machines and running into discrepancies in our yarn setups. Do you know why it needs to be removed in order to fix the broken links to `/doc`? Or were there other errors that this change addresses?

Reply:
When we try to run yarn install with this line, we get
```bash
Internal Error: ENOENT: no such file or directory, stat '/Users/daniilankusin/monomer/doc/.yarn/releases/yarn-4.3.1.cjs'
Error: ENOENT: no such file or directory, stat '/Users/daniilankusin/monomer/doc/.yarn/releases/yarn-4.3.1.cjs'
```
Josh asked [the same question](https://github.com/polymerdao/monomer/pull/121#discussion_r1714288913)
I don't have this directory when I follow the instructions.

Reply:
Ahh this makes sense. That path does not exist *until after* `yarn install` is run. I think instead of removing it we should include the release in version control by modifying [the `.gitignore`](https://github.com/polymerdao/monomer/blob/main/doc/.gitignore) and then committing the `.yarn/releases/` directory. See [the yarn docs](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored) for the rationale behind this:

> _`.yarn/releases` contain the Yarn releases used in the current repository (as defined by [yarn ](https://yarnpkg.com/cli/set/version)[set version](https://yarnpkg.com/cli/set/version)). You will want to keep them versioned (this prevents potential issues if, say, two engineers use different Yarn versions with different features)._

Reply:
Thanks @phasewalk1. To make sure I'm following, we'll want to use the non-zero installs method from the docs:

```
.pnp.*
.yarn/*
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```
Is that right?

Reply:
Yes that's correct 👍🏻 @joshklop

Reply:
All good, thanks